egs/lrs/avsr1/local/training/finetuneav/attention.py

```diff
@@ -0,0 +1,76 @@
+import math
```

Comment:
There are some overlapped functions within the PR (sorry that I can not find all of them because of the large PR)

Reply:
Do you mean attention.py is overlapped with espnet/nets/pytorch_backend/transformer/attention.py?  If yes, please see my comment below :)